proto/chainlink/v1beta/tx.proto

```diff
@@ -50,6 +50,7 @@ message MsgFeed {
   // Module owner who signs the add feed tx
   bytes moduleOwnerAddress = 7 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
   // TODO: RewardConfig: Struct describing how rewards should be paid
+  uint32 feedReward = 8;
```

Comment:
I assume that the reason why type here is `uint32` is because the reward config is just a number of amount? something like all the data providers would get some amount a token. However, I think this `feedReward` should be a complex structure which might include but not limit to 1)reward calculation strategy which could be based on data accuracy, 2)participation rate, 3)either submitter or just regular signer and so on. It's better to make feedReward config as open as possible to incentivize data provider. Simply config it as a number might not be a good idea here.

Reply:
Correct, I will add distribution ot all dp's, missed that.
also,
1.) i dont think it's within our scope to create the data accuracy logic nor the calculation strategy. The calc should be based on just the data provider's submission (afaik) and the feed reward set by the feed owner.
2.) what do you propose is the `pariticipation rate`?
3.) yes, we will need to compensate for gas. i think i can just query the transaction hash and grab the gas that the submitter paid to compensate additional tokens there

the `feedReward` can be set by the `feedOwner` by using the tx, similar to the way we change the `deviation` `heartbeat` and `submissioncount`. will add that to this PR as well!

Reply:
I understand that feed owner needs to add/modify/remove feed reward by a tx. 

1.something might not be in our scope for sure, but it's ok to put a scalable design now, right? we don't need to impl the concrete func for `data accuracy` now, maybe not `pariticipation rate` either, but we could design an interface to make it all possible.

2.`pariticipation rate` is a reward cal strategy what based on data provider's submit frequency, e.g: A only submit his observation like once a month, however it should be once a day

3. ok